docs/pages/v2.md

```diff
@@ -781,93 +781,10 @@ module.exports = {
 };
 ```
 
-## remix dev
```

Comment:
I think we should keep this v2 migration guide in place as-is through the release of v2 (and likely for a long while after).  Folks are going to still be on 1.x for a while and will want access to a centralized migration guide to get to v2.